examples/python/model-generate.py

```diff
@@ -0,0 +1,43 @@
+import onnxruntime_genai as og
+import argparse
+
+def main(args):
+    print("Loading model...")
+    model=og.Model(f'{args.model}', og.DeviceType.CPU if args.execution_provider == 'cpu' else og.DeviceType.CUDA)
+    print("Model loaded")
+    tokenizer=og.Tokenizer(model)
+    tokenizer_stream=tokenizer.create_stream()
+    print("Tokenizer created")
+
+    if args.input is not None:
+        prompt = args.input
```

Comment:
could we make prompt an array?

Reply:
should the user be able to input multiple prompts somehow?